src/test/java/Playlists.java

```diff
@@ -0,0 +1,101 @@
+
+        LoginPage loginPage = new LoginPage(getDriver());
+        HomePage homePage = new HomePage(getDriver());
+        BasePage basePage = new BasePage(getDriver());
+        basePage.navigateToPage(homeUrl);
+        loginPage.loginWithValidCredentials(email, password);
+
+        System.out.println("Message should be: " + msg);
+        String msgReturn = homePage.createThePlaylist(Playlist);
+        Assert.assertTrue(msgReturn.contains(msg));
+    }
+
+    @Test (dependsOnMethods = {"createPlaylist"})
+    @Parameters ({"Playlist"})
+    public void deletePlaylist (String Playlist){
+
+        //variables
+        String msg = "Deleted playlist \"" + Playlist + ".\"";
+
+        LoginPage loginPage = new LoginPage(getDriver());
+        HomePage homePage = new HomePage(getDriver());
+        BasePage basePage = new BasePage(getDriver());
+        basePage.navigateToPage(homeUrl);
+        loginPage.loginWithValidCredentials(email, password);
+
+        homePage.selectPlaylist(Playlist);
+        System.out.println("Message should be: " + msg);
+        String msgReturn = homePage.deleteSelectedPlaylist();
+        Assert.assertTrue(msgReturn.contains(msg));
+    }
+
+    @Test
+    @Parameters({"PlaylistToRename", "NewPlaylistName"})
+    public void renamePlaylist (String PlaylistToRename, String NewPlaylistName) {
+
+        //variables
+        String msg = "Updated playlist \"" + NewPlaylistName + ".\"";
+
+        LoginPage loginPage = new LoginPage(getDriver());
+        HomePage homePage = new HomePage(getDriver());
+        BasePage basePage = new BasePage(getDriver());
+        basePage.navigateToPage(homeUrl);
+        loginPage.loginWithValidCredentials(email,password);
+
+        System.out.println("Message should be: " + msg);
+        String msgReturned = homePage.updatePlaylist(PlaylistToRename,NewPlaylistName);
+        Assert.assertEquals(msgReturned,msg);
+    }
+
+    @Test
+    @Parameters ({"SongToSearch", "PlaylistToAddSong"})
+    public void addSongToPlaylist (String song, String playlist) {
+
+        //variables
+        String msg = "Added 1 song into";
+
+        LoginPage loginPage = new LoginPage(getDriver());
+        HomePage homePage = new HomePage(getDriver());
+        BasePage basePage = new BasePage(getDriver());
+        basePage.navigateToPage(homeUrl);
+        loginPage.loginWithValidCredentials(email, password);
+        String msgReturn = homePage.searchSong(song).viewSearchedSong().clickAddToPls(playlist);
+        Assert.assertTrue(msgReturn.contains(msg));
+    }
+
+//    @Test
+//    public void playSong () {
+//        LoginPage loginPage = new LoginPage(getDriver());
+//        HomePage homePage = new HomePage(getDriver());
+//        BasePage basePage = new BasePage(getDriver());
+//        basePage.navigateToPage(homeUrl);
+//        loginPage.loginWithValidCredentials(email, password);
+//        Assert.assertTrue(homePage.playNextSong().isPlaying());
+//    }
```

Comment:
**I recommend not using code comments:** for, example, when you have a code that cannot be understood, then comments are not the solution. Try to rewrite the code or rename fields and other elements in a way that the reader will understand the action that you are doing there.

Reply:
Got it, same comment as before =)